ekncontent/ekncontent/eknc-domain.c

```diff
@@ -891,7 +938,7 @@ eknc_domain_query_finish (EkncDomain *self,
   QueryState *state = g_task_get_task_data (task);
   return g_object_new (EKNC_TYPE_QUERY_RESULTS,
                        "upper-bound", state->upper_bound,
-                       "models", state->models,
+                       "models", ptr_array_to_list_shallow (state->models),
```

Comment:
No, we're not making any API guarantees other than "it will be really inconvenient if the way it's accessed from JS changes" :smile:

However, I'm not sure if gobject-introspection supports introspecting the type of a GPtrArray-valued property, whereas with a GList it does.